### PR TITLE
[4.0] Add edit button to remaining list views

### PR DESCRIPTION
**Pull request by C-Lodder**
Pull Request for Issue #17226

### Summary of Changes

Adds an edit icon to the list items in `com_languages` as per the other components

### CODE REVIEW

File: administrator/components/com_languages/tmpl/languages/default.php [hunk 1]
```diff
@@ -118,7 +118,9 @@
 								<td>
 									<span class="editlinktip hasTooltip" title="<?php echo JHtml::_('tooltipText', JText::_('JGLOBAL_EDIT_ITEM'), $item->title, 0); ?>">
 									<?php if ($canEdit) : ?>
-										<a href="<?php echo JRoute::_('index.php?option=com_languages&task=language.edit&lang_id=' . (int) $item->lang_id); ?>"><?php echo $this->escape($item->title); ?></a>
+										<?php $editIcon = '<span class="fa fa-pencil-square mr-2" aria-hidden="true"></span>'; ?>
```
Comment (zero-24) [inline administrator/components/com_languages/tmpl/languages/default.php]: why does this need to be a PHP Variable? Can't we use it as direct html?

File: administrator/components/com_languages/tmpl/overrides/default.php [hunk 2]
```diff
@@ -87,7 +87,9 @@
 								</td>
 								<td>
 									<?php if ($canEdit) : ?>
-										<a id="key[<?php echo $this->escape($key); ?>]" href="<?php echo JRoute::_('index.php?option=com_languages&task=override.edit&id=' . $key); ?>"><?php echo $this->escape($key); ?></a>
+										<?php $editIcon = '<span class="fa fa-pencil-square mr-2" aria-hidden="true"></span>'; ?>
```
Comment (zero-24) [inline administrator/components/com_languages/tmpl/overrides/default.php]: same as above applys

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 5e3356301c6c5d70174666657b35cb6e44cd1e48<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/17242">issues.joomla.org/tracker/joomla-cms/17242</a>.</sub>

File: administrator/components/com_languages/tmpl/languages/default.php (hunk 1 above)
Comment (C-Lodder) [reply administrator/components/com_languages/tmpl/languages/default.php]: My bad. It was a ternary operator in other components. Fixed now

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on c4a48a6e26fe85eb95e479eb49645c61f530618c<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/17242">issues.joomla.org/tracker/joomla-cms/17242</a>.</sub>

**Comment by dgrammatiko** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/17242">issues.joomla.org/tracker/joomla-cms/17242</a>.</sub>

**Comment by C-Lodder** [comment]
please wait. Just doing the rest of the components

**Comment by dgrammatiko** [comment]
I won't remove the RTC, just ask @brianteeman to confirm when you're done ;)

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 6bac151db508b87daa6d3e7594d05aae89ca968a<br /><br />retested with the full list - all looks good<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/17242">issues.joomla.org/tracker/joomla-cms/17242</a>.</sub>

**Comment by ghost** [comment]
Click on "Apply Patch" got:
bildschirmfoto 2017-07-25 um 15 27 10

**Comment by C-Lodder** [comment]
@franz-wohlkoenig - Re-fetch the latest code from the repo before applying the PR. Another PR was merged whilst I was doing this which changed the file location.

**Comment by ghost** [comment]
Reinstalled core Files per Joomla-Update + Fetched Data in Patch Tester, same Error-Message.

**Comment by C-Lodder** [comment]
No idea then, sorry. I don't use the patch tester

**Comment by mbabker** [comment]
If you're using the nightly builds those are going to be some commits behind until the next run (they are nightly builds after all, not generated on every commit to our dev branches).

**Comment by ghost** [comment]
@mbabker thanks, have forgotten one more Time this Difference. Is there a url like for nightly builds to update on Dev-Branche?

**Comment by brianteeman** [comment]
https://github.com/joomla/joomla-cms/archive/staging.zip

**Comment by brianteeman** [comment]
https://github.com/joomla/joomla-cms/archive/4.0-dev.zip

**Comment by ghost** [comment]
thanks @brianteeman, using this url got:
bildschirmfoto 2017-07-25 um 15 56 32

**Comment by zero-24** [comment]
@franz-wohlkoenig you can not use that as update server ;) You need to download the link shared by brian and use upload and install ;)

**Comment by ghost** [comment]
@zero-24 thanks for helping. After "Upload & Update" > "Upload & Install" zip-File and new "Fetch Data" in Patch Tester got same Message.

**Pull request merged**